runtime/src/staking.rs

```diff
@@ -651,6 +655,8 @@ decl_storage! {
         /// The map from (wannabe) validators to the status of compliance
         pub PermissionedValidators get(permissioned_validators):
             linked_map T::AccountId => Option<PermissionedValidator>;
+        /// No of seconds allowed as a tradeoff in the kyc expiry check
+        pub KYCExpiryTradeOff get(kyc_expiry_tradeoff) config(): u64;
```

Comment:
Lets use the bonding duration as the KYCExpiryTradeOff so that nominators are guaranteed to be valid until the end of their bonding period.